packages/create-pantheon-decoupled-kit/src/templates/next-drupal/components/layout.jsx.ts

```diff
@@ -0,0 +1,6 @@
+import { layoutTemplate } from '@cli/templates/partials/nextjs-shared/layoutT';
+import { TemplateFn } from '@cli/types';
+
+const jsx: TemplateFn = () => /* jsx */ `${layoutTemplate(false, 'drupal')}`;
```

Comment:
```suggestion
const jsx: TemplateFn = ({ data }) => /* jsx */ `${layoutTemplate({search: data.search, cmsType: data.drupal || data.wp })}`;
```
Use the `data` instead of explicit args. Also use named args on the templates if there is > 1

Reply:
Err actually the `data.drupal || data.wp` won't really work since those are booleans and we aren't passing those as strings anywhere. Maybe there's some trick we can do with the `CMSType` and our `Input` type.